op-node/flags/flags.go

```diff
@@ -14,10 +14,10 @@ import (
 
 // Flags
 
-const envVarPrefix = "OP_NODE_"
+const envVarPrefix = "OP_NODE"
 
 func prefixEnvVar(name string) string {
-	return envVarPrefix + name
+	return envVarPrefix + "_" + name
 }
```

Comment:
We could instead use `op-service.PrefixEnvVar` here, like for the batcher and proposer.

Reply:
We could. That's a bigger change though. The batcher/proposer also don't have a local `prefixEnvVar` function.

Reply:
Just thought now that you touch this, might as well delete the code duplication. Up to you.

Reply:
TBH this is minor enough + something we don't really want to change that I don't mind having it duplicated.